lib/mandrill-rails/web_hook_processor.rb

```diff
@@ -34,8 +34,8 @@ module Mandrill::Rails::WebHookProcessor
   extend ActiveSupport::Concern
 
   included do
-    skip_before_filter :verify_authenticity_token
-    before_filter :authenticate_mandrill_request!, :only => [:create]
+    skip_before_action :verify_authenticity_token
+    before_action :authenticate_mandrill_request!, :only => [:create]
```

Comment:
@spastorino I like that idea. I just merged all the rails 5 efforts on master and took the easy way out "1.5.0+ only supports rails 5". If someone would like to try and bring back backwards compatibility, I'd really appreciate that.